proto/topodata.proto

```diff
@@ -323,4 +323,7 @@ message CellInfo {
   // Root is the path to store data in. It is only used when talking
   // to server_address.
   string root = 2;
+
+  // Region is a group this cell belongs to
```

Comment:
Please be a bit more specific in this comment (and end it with a period). Something like:

// Region is a group this cell belongs to. Used by vtgate to route traffic to
// other cells when there is no available tablet in the current region.